var/spack/repos/builtin/packages/py-metpy/package.py

```diff
@@ -11,9 +11,10 @@ class PyMetpy(PythonPackage):
     with weather data."""
 
     homepage = "https://github.com/Unidata/MetPy"
-    pypi     = "MetPy/MetPy-1.0.tar.gz"
+    pypi     = "MetPy/MetPy-1.0.1.tar.gz"
+    maintainers = ['dopplershift']
 
-    version('1.0', sha256='11b043aaa4e3d35db319e96bb9967eb9f73da653e155bca2d62f838108b100dc')
+    version('1.0.1', sha256='16fa9806facc24f31f454b898741ec5639a72ba9d4ff8a19ad0e94629d93cb95')
```

Comment:
Unlike other package managers, Spack is designed to allow users to install multiple versions of software. So instead of removing 1.0, I would keep both versions. For the dependencies, you can do something like:
```python
depends_on('py-pandas@0.24.0:', when='@1.0.1:', type=('build', 'run'))
depends_on('py-pandas@0.22.0:', when='@1.0', type=('build', 'run'))
```
If you don't want to maintain 1.0 anymore, you can add `deprecated=True` and remove it in the next Spack release.

Reply:
Ah, ok.